src/app/frontend/logs/logs.html

```diff
@@ -51,47 +51,58 @@
     <div class="kd-logs-style-buttons">
       <md-button class="kd-logs-toolbar-button"
                  ng-click="ctrl.onTextColorChange()">
-        <md-icon md-font-library="material-icons"
-                 ng-class="ctrl.getColorIconClass()">
+        <md-icon ng-class="{'kd-logs-color-icon-invert': ctrl.logsService.getInverted(),
+                            'kd-logs-color-icon': !ctrl.logsService.getInverted()}">
           format_color_text
         </md-icon>
       </md-button>
       <md-button class="kd-logs-toolbar-button"
                  ng-click="ctrl.onFontSizeChange()">
-        <md-icon md-font-library="material-icons"
-                 ng-class="ctrl.getSizeIconClass()">
+        <md-icon ng-class="{'kd-logs-size-icon-compact': ctrl.logsService.getCompact(),
+                            'kd-logs-size-icon': !ctrl.logsService.getCompact()}">
           text_fields
         </md-icon>
       </md-button>
       <md-button class="kd-logs-toolbar-button"
                  ng-click="ctrl.onShowTimestamp()">
-        <md-icon md-font-library="material-icons"
-                 class="kd-logs-icon">
-          {{ctrl.getTimestampIcon()}}
+        <md-icon class="kd-logs-icon">{{ctrl.logsService.getShowTimestamp() ? 'timer' : 'timer_off'}}</md-icon>
+      </md-button>
+      <md-button class="kd-logs-toolbar-button"
+                 ng-click="ctrl.toggleLogFollow()">
+        <md-icon class="kd-logs-icon">
+          refresh
+          <svg class="kd-cross-style"
+               ng-if="!ctrl.logsService.getFollowing()"
+               width="32px"
+               height="32px"
+               viewBox="0 0 32 32">
+            <line class="kd-cross-line-white" x1="9" x2="27" y1="9" y2="27"/>
+            <line class="kd-cross-line-black" x1="9" x2="27" y1="10" y2="28"/>
+          </svg>
         </md-icon>
+        <md-tooltip md-delay="500"
+                    md-autohide>
+          [[Toggle auto-refresh (every {{ctrl.refreshInterval / 1000}} seconds)|]]
```

Comment:
Description missing?

Reply:
Oh, I did not notice it before merging (I was already on the page). We do not need description everywhere. Translators will see English entry anyways.